src/styles/index.scss

```diff
@@ -1,3 +1,11 @@
+@import '../../src/styles/variables';
+@import '../../src/styles/card';
+@import '../../src/styles/stars';
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
```

Comment:
You can remove this, because 400 is default value in browser

Reply:
> You can remove this, because 400 is default value in browser

Not fixed